lib/Ohmbrewer_Pump.cpp

```diff
@@ -61,6 +63,7 @@ int Ohmbrewer::Pump::doWork() {
  * @returns The time taken to run the method
  */
 int Ohmbrewer::Pump::doDisplay(Ohmbrewer::Screen *screen) {
+    screen->print("Pump");
```

Comment:
Agreed over Hangouts that the display refactoring should go into a new Issue/PR.